docs/references/admin_server.rst

```diff
@@ -54,3 +54,45 @@ Metrics
 =======
 
 Provides :ref:`metrics`.
+
+Debugging
```

Comment:
Not sure if it's the best title. I remember it was mentioned somewhere that `--dump-config` and `--dump-schema` were used for debugging purposes.

Reply:
I think we should solve it like this.

- Remove the Healthcheck subtitle with its paragraph
- Put the Important note under Live
- Remove this Debugging title.

Reply:
Ok, there's another option. Just remove this Debugging title and rename the new subtitles according to the below suggestions

Reply:
Seems good. I modified it accordingly.